Travel agency

williadav opened the pull request.

paul673 reviewed (commented):
1. Reset to alphabetical sorting when clicking "Land" (reisebrev/)'
2. It is hard to find the travelletters on the admin page. (Testet for 1000). Solutions:
     -   Introduce a edit button on the detail page of a travelletter
     -   Introduce sorting on the admin page
 3. Styling of the detail page :)

williadav commented:
1. Added alphabetical sorting
2. Added edit button on detail page, the admin page sorting can be looked into later
3. Changed background color of information box on detail page

paul673 reviewed (approved):
:)
